renderers/chords_renderer.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+const parseVerse = require('../verse.js')['parseVerse'];
+
+class ChordsRenderer {
+  constructor(chordChartJson) {
+    this.chordChartJson = chordChartJson;
+  }
+
+  createHtmlChordChart() {
+    const chordChartHtml = document.createElement('div');
+    chordChartHtml.className = 'chart';
+
+    this.chordChartJson.forEach((verse) => {
+      const verseDiv = document.createElement('div');
+      verseDiv.className = 'verse';
+
+      this.createListOfWrappedVoices(verse)
+        .forEach((event) => {
+          verseDiv.appendChild(event);
+        });
+
+      chordChartHtml.appendChild(verseDiv);
+    });
+
+    return chordChartHtml;
+  }
+
+  createListOfWrappedVoices(verse) {
+    // Assumes voices are sorted by index and grouped by voice.
+    const voiceElements = new Map();
```

Comment:
This makes me think that our intermediate format that we came up with in our design doc was not optimal. Maybe the structure of a phrase should have been:

```
{
  'c1': [
      {index: 0, content: new Chord('G')},
      {index: 9, content: new Chord('E', 'm')},
      {index: 33, content: new Chord('B', 'm')},
  ],
  'l1': [
      {index: 0, content: 'I'},
      {index: 2, content: 'didn\'t'},
      {index: 9, content: 'ask,'},
      {index: 14, content: 'you'},
      {index: 18, content: 'shouldn\'t'},
      {index: 28, content: 'have'},
      {index: 33, content: 'told'},
      {index: 38, content: 'me'},
  ]
}
```

If you agree, add a TODO and create an issue to fix this.

Reply:
Yeah, I like this structure more. Much more explicit instead of assuming the grouping. I'll add a TODO here and in verse.js.

Reply:
For some clarity if anyone comes to read this later: ordering by just index means we don't preserve the order it was written in. If we group and order by voice, we can maintain the original markdown order.